tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml

```diff
@@ -1,6 +1,6 @@
 <tool id="NSPDK_candidateClust" name="NSPDK_candidateClusters" version="9.2">
 	<requirements>
-		<requirement type="package" version="0.1">graphclust-wrappers</requirement>
```

Comment:
In some cases, though rare, we may need to keep some wrappers with older version..